copier.yml

```
@@ -150,7 +150,7 @@ copier__operating_system:
   choices:
     k3s: "k3s"
     talos: "talos"
-  help: "Which operating system to use?"
+  help: "The lightweight Kubernetes distribution to use."
```

Talos is not that lightweight. It's a full k8s distro.
```suggestion
  help: "Deploy Kubernetes on k3s running on Ubuntu or on Talos Linux."
```